app/controllers/dashboard_controller.rb

```diff
@@ -29,33 +30,11 @@ class DashboardController < ApplicationController
   # fields that customer uses) and also skips the redirection to the setup process (when a worker
   # asks for the autoyast profile we will either serve it, or return an error).
   def autoyast
-    @controller_node = Pillar.value pillar: :dashboard
+    autoyast = AutoyastValues.new
```

Comment:
we might not need to cache this in a variable do we? `pollute_variables!(AutoyastValues.new)`